src/unix/bsd/netbsdlike/netbsd/x86_64.rs

```diff
@@ -21,3 +22,44 @@ pub const PT_GETREGS: ::c_int = PT_FIRSTMACH + 1;
 pub const PT_SETREGS: ::c_int = PT_FIRSTMACH + 2;
 pub const PT_GETFPREGS: ::c_int = PT_FIRSTMACH + 3;
 pub const PT_SETFPREGS: ::c_int = PT_FIRSTMACH + 4;
+
+s_no_extra_traits! {
+    #[repr(align(8))]
+    pub struct mcontext_t {
+        pub __gregs: [c___greg_t; 26],
+        pub _mc_tlsbase: c___greg_t,
+        pub __fpregs: [::c_char; 512],
```

Comment:
So, is there a reason to use `s_no_extra_traits!`? We can use this hack for the auto-derive issue:

https://github.com/rust-lang/libc/blob/de9842278fc205166ccc067ff16f5651466a2e0c/src/unix/linux_like/android/b64/aarch64/align.rs#L25-L27